db/seeds.rb

```diff
@@ -3,5 +3,47 @@
 #
 # Examples:
 #
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+
+categories = [
+  	{ name: 'Italian' }, 
+  	{ name: 'Southwestern' }, 
+  	{ name: 'Sushi' },
+  	{ name: 'Burgers' },
+  	{ name: 'Pizza' },
+  	{ name: 'Wings' },
+  	{ name: 'Indian' },
+  	{ name: 'Tapas' },
+  	{ name: 'Dim Sum' },
+  	{ name: 'Fast Food' },
+  	{ name: 'Food Truck' },
+  	{ name: 'American' },
+  	{ name: 'French' },
+  	{ name: 'Pho' },
+  	{ name: 'Mediterranean' },
+  	{ name: 'Subs' }
+]
+
+categories.each do |x|
+	Category.create(x)
+end
```

Comment:
I know Im being a stickler, but please try to use representative variable names. Replace `x` with something like `category`